IdeasTracker/IdeasTracker/View/IdeasView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  ContentView.swift
+//  IdeasTracker
+//
+//  Created by Wen Yu Ge on 2024-04-19.
+//
+
+import SwiftUI
+import AppwriteModels
+
+struct IdeasView: View {
+    @EnvironmentObject private var IdeasViewModel: IdeasViewModel
```

Comment:
This can be a `@StateObject` that is instantiated here, since this view should own it and manage it's lifecycle

Reply:
Done